packages/rum/README.md

```diff
@@ -342,6 +342,68 @@ A service is an independent, deployable code repository that maps to a set of pa
 - If your browser application was constructed as a monolith, your RUM application has one service name for the application.
 - If your browser application was constructed as separate repositories for multiple pages, edit the default service names throughout the lifecycle of your application.
 
+### Access internal context
+
+After the Datadog browser RUM SDK is initialized, you can access the internal context of the SDK. This allows you to access:
+
+| Attribute      | Description                                                   |
+| -------------- | ------------------------------------------------------------- |
+| application_id | ID of the application                                         |
+| session_id     | ID of the session                                             |
+| user_action    | Object containing action ID (or undefined if no action found) |
+| view           | Object containing details about the current view event        |
+
+More details can be found on the [**RUM Browser Data Collected** page][2].
+
+#### Example
+
+```
+{
+  application_id : "xxx",
+  session_id : "xxx",
+  user_action: { id: "xxx" },
+  view : {
+    id : "xxx",
+    referrer : "",
+    url: "http://localhost:8080/"
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏
```suggestion
    url: "http://localhost:8080/",
    name: "homepage"
```